src/commands/operator/faucet.ts

```diff
@@ -0,0 +1,51 @@
+import { Command } from "@oclif/core";
+import axios, { AxiosError } from "axios";
+import { isAddress } from "ethers/lib/utils";
+
+export default class OperatorFaucet extends Command {
+  static description = "Request test tokens for node operator (0.2 Sepolia ETH and 100 EARTHFAST tokens)";
+
+  static args = [
+    {
+      name: "address",
+      required: true,
+      description: "Ethereum address to receive tokens",
+    },
+  ];
+
+  async run(): Promise<void> {
+    const { args } = await this.parse(OperatorFaucet);
+    const { address } = args;
+
+    // Validate ethereum address
+    if (!isAddress(address)) {
+      this.error("Invalid Ethereum address provided");
+    }
+
+    try {
+      this.log(`Sending request to faucet for address: ${address}`);
+
+      const response = await axios.post(`https://jorge-server.earthfast-dev.com/operator-faucet`, {
```

Comment:
update url. also we should correctly POST to staging or testnet server based on network